src/browser/api_protocol/transport_strategy/elipc_strategy.ts

```diff
@@ -100,33 +102,33 @@ export class ElipcStrategy extends ApiTransportBase<MessagePackage> {
     }
 
     private canTrySend(routingInfo: any): boolean {
-        const { browserWindow, frameRoutingId } = routingInfo;
-        const browserWindowLocated = browserWindow;
-        const browserWindowExists = !browserWindow.isDestroyed();
+        const { webContents, frameRoutingId } = routingInfo;
+        const webContentsLocated = webContents;
+        const webContentsExists = !webContents.isDestroyed();
         const validRoutingId = typeof frameRoutingId === 'number';
-        return browserWindowLocated && browserWindowExists && validRoutingId;
+        return webContentsLocated && webContentsExists && validRoutingId;
     }
 
     // Dispatch a message
     private innerSend(payload: string,
                       frameRoutingId: number,
                       mainFrameRoutingId: number,
-                      browserWindow: any): void {
+                      webContents: WebContents): void {
         if (frameRoutingId === mainFrameRoutingId) {
             // this is the main window frame
             if (coreState.argo.framestrategy === 'frames') {
```

Comment:
Lets make the default `webContents.sendToFrame(frameRoutingId, channels.CORE_MESSAGE, payload);` with the only way to opt into the shared name `enableDeprecatedSharedName` as seen [here](https://github.com/HadoukenIO/core/blob/develop/src/browser/api_protocol/transport_strategy/elipc_strategy.ts#L188).

Reply:
`coreState.argo.framestrategy` can die